### ISSUE TITLE
Is there something like an ItemDictionary?

### ISSUE THREAD

**Issue by bbb0x** [Grover-c13/PokeGOAPI-Java#742]
I use PokeDictionary to output the pokemon names, but I can not find such a class for items and PokeDictionary only accepts Pokemon input. I know it wouldn't be much of a work to write this class myself but I'm just curious if I'm just not finding it?

**Comment by fabriciorby** [comment]
I'm curious about this too, it would be nice.
If someone have already written one please let us know :)

**Comment by bbb0x** [comment]
This is what I wrote very quickly but english only:

```
import java.util.Locale;
import POGOProtos.Inventory.Item.ItemIdOuterClass.ItemId;

public class ItemDictionary {

    public static String getDisplayName(ItemId id, Locale locale) throws Exception
    {
        if(locale != locale.ENGLISH)
            throw new Exception("Only english supported.");

        switch(id)
        {
        case ITEM_POKE_BALL:
        return "Pokeball";
        case ITEM_GREAT_BALL:
            return "Greatball";
        case ITEM_ULTRA_BALL:
            return "Ultraball";
        case ITEM_MASTER_BALL:
            return "Masterball";
        case ITEM_POTION:
            return "Potion";
        case ITEM_SUPER_POTION:
            return "Super Potion";
        case ITEM_HYPER_POTION:
            return "Hyper Potion";
        case ITEM_REVIVE:
            return "Revive";
        case ITEM_MAX_REVIVE:
            return "Max Revive";
        case ITEM_RAZZ_BERRY:
            return "Razz Berry";
        case ITEM_BLUK_BERRY:
            return "Bluk Berry";
        case ITEM_NANAB_BERRY:
            return "Nanab Berry";
        case ITEM_PINAP_BERRY:
            return "Pinap Berry";
        case ITEM_WEPAR_BERRY:
            return "Wepar Berry";
        case ITEM_LUCKY_EGG:
            return "Lucky Egg";
        case ITEM_INCENSE_COOL:
            return "Incense Cool";
        case ITEM_INCENSE_FLORAL:
            return "Incense Floral";
        case ITEM_INCENSE_ORDINARY:
            return "Incense Ordinary";
        case ITEM_INCENSE_SPICY:
            return "Incense Spicy";
        case ITEM_INCUBATOR_BASIC:
            return "Incubator";
        case ITEM_INCUBATOR_BASIC_UNLIMITED:
            return "Incubator (Unlimited)";
        case ITEM_ITEM_STORAGE_UPGRADE:
            return "Storage Upgrade";
        case ITEM_SPECIAL_CAMERA:
            return "Camera";
        case ITEM_TROY_DISK:
            return "Troy Disk";
            default:
                return "Unknown Item";

        }
    }
}
```

**Comment by yutamago** [comment]
@bbb0x @fabriciorby 
I fixed this issue in my PR #752. Feel free to try it out and suggest changes.
And if you happen to know any of the missing translations, feel free to leave them in a comment.

**Comment by fabriciorby** [comment]
Thank you guys :)
